card/card.gemspec

```diff
@@ -34,8 +34,9 @@ Gem::Specification.new do |s|
 
     [ 'smartname',    '0.2.3'   ],
 
+    [ 'activerecord-session_store'],
     [ 'uuid',         '~> 2.3'  ],
-    [ 'paperclip',    '~> 2.8'  ],
+    [ 'paperclip',    '~> 4.2'  ],
```

Comment:
no, that's there because ActiveJob needs Rails 4 and Rails 4 needs that paperclip update.